src/utils/FileManager.js

```diff
@@ -231,7 +231,8 @@ class FileManager {
     }
     const { filePath, relPath, moduleName } = file;
     const isModConf = filePath.indexOf('conf') >= 0 && moduleName;
-    let destPath = path.join(this.buildDir, relPath);
+    const rPath = relPath.replace(this.srcDir, ''); // Sometimes relPath returns full path to source dir
```

Comment:
Interesting! Would `path.relative(this.srcDir, relPath)` work? I'm hoping to avoid the case where the slashes don't match (so if `this.srcDir` was `C:\test\test.js` and `relPath` was `C:/test/test.js` you cannot do a simple string replace).